### DISCUSSION TITLE
Support for Workload Identity required from a Kubernetes Linux POD.

### DISCUSSION THREAD

**Discussion by cloudtidings** [sqlalchemy/sqlalchemy#9605] [Ideas]
Support for Workload Identity required. https://github.com/Azure/azure-workload-identity

Is your feature request related to a problem? Please describe.
There's no clear way currently to connect to Azure SQL using Workload Identity with Microsoft's Managed Identities from a Kubernetes Linux POD. 

Describe the solution you'd like
I'd either like this strategy to be integrated into sqlalchemy somehow or for it to be included in the documentation so other people don't have to spend their time figuring this out too.

**Comment by CaselIT** [comment]
Hi,

I don't think this question is for sqlalchemy honestly. Sqlalchemy has no affiliation with Microsoft or Azure.

There are various examples of using a dynamic token or url string. Figure out how to get the url string/secret from python and you can use one of those examples, like https://docs.sqlalchemy.org/en/20/dialects/mssql.html#connecting-to-databases-with-access-tokens
